downstream/modules/platform/con-aap-upgrade-planning.adoc

```diff
@@ -7,34 +7,20 @@
-* If you need to upgrade {RHEL} and {ControllerName}, you must first backup and restore your {ControllerName} data.
-* Clustered upgrades require special attention to instance and instance groups before upgrading.
+* See link:{URLPlanningGuide}/platform-system-requirements[System requirements] in the {TitlePlanningGuide} guide to ensure you have the topologies that fit your use case. 
++
+[NOTE]
+====
+2.4 to 2.5 upgrades now include link:{URLPlanningGuide}/ref-aap-components#con-about-platform-gateway_planning[Platform gateway]. Ensure you review the 2.5 link:{URLPlanningGuide}/ref-network-ports-protocols_planning[Network ports and protocols] for architectural changes.
+====
++
+* Ensure you have a valid subscription from a previous version. You must provide your credentials or a link:https://access.redhat.com/articles/5807761[subscriptions manifest] upon upgrading to the latest version of {PlatformNameShort}.
+* Clustered upgrades require special attention to instance and instance groups before upgrading. Ensure you capture your inventory or instance group details before upgrading.
+* Upgrading 2.4 {EDAName} is not supported.[ADD FURTHER TEXT HERE]
 
 [role="_additional-resources"]
 .Additional resources
-* link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_controller_user_guide/controller-managing-subscriptions#controller-importing-subscriptions[Importing a subscription]
-* link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_controller_administration_guide/controller-backup-and-restore[Backup and restore]
-* link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_controller_administration_guide/controller-clustering[Clustering]
-
-[discrete]
-== {HubNameStart}
-
-* When upgrading to {PlatformNameShort} {PlatformVers}, you can either add an existing {HubName} API token or generate a new one and invalidate any existing tokens.
-* Existing container images are removed when upgrading {PlatformNameShort}. 
-This is because, when upgrading {PlatformNameShort} with `setup.sh` script, podman `system reset -f` is executed. 
-This removes all container images on your {PlatformNameShort} nodes then pushes the new {ExecEnvShort} image that is bundled with installer.
-See xref:proc-running-setup-script-for-updates[Running the {PlatformName} installer setup script].
-
-[role="_additional-resources"]
-.Additional resources
-* <<editing-inventory-file-for-updates_{context}, Setting up the inventory file >>
-
-[discrete]
-== {EDAcontroller}
-//ATTENTION: Remove this section for EDA 1.0.4; customers will no longer need to perform deactivation because services will be automatically restored after upgrade and migration. 
+* link:{URLCentralAuth}/assembly-gateway-licensing#proc-attaching-subscriptions[Attaching a subscription]
+* xref:platform/con-backup-aap.adoc[Backup and restore]
+* link:{URLControllerAdminGuide}/controller-clustering[Clustering]
```

Comment:
A fourth bullet is missing, "Planning your installation"

Reply:
Good catch! Should be fixed now.